keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -739,49 +674,42 @@ providers such as Okta, Azure AD, etc:
 
 #### Beta
 
-- Gather feedback
 - Complete benchmarks
+  -  Benchmarks are required to see how different CEL expressions affects authentication time.
+     -  There will be a upper bound of 5s for the CEL expression evaluation. We will use the `apiserver_authentication_latency_seconds` metric to monitor this.
 - Add metrics
-- Initial e2e test with an external provider completed and enabled
+- Support > 1 JWT authenticator and > 1 audiences
+- Enable automatic reload of the configuration
+- Add tests
+  - Tests for automatic reload of the configuration
 
 #### GA
 
+- Gather feedback
 - Add a full documentation with examples for the most popular providers, e.g., Okta, Dex, Auth0
 - Migration guide
-- Deprecation warnings for non-structured OIDC provider configuration
+- e2e test with an external provider completed and enabled
+- Get distributed claims working with CEL
+- Decide if we want to support egress selection configuration and how to do so
+
+### Upgrade / Downgrade Strategy
 
-#### Deprecation
+While the feature is in Alpha, there is no change if cluster administrators want to
+keep on using command line flags.
 
-kube-apiserver `--oidc-*` flags require deprecation warnings on the stable release of the feature.
+When the feature goes to Beta/GA or the cluster administrators want to configure
+jwt authenticators using the configuration file, they need to make sure:
 
-### Upgrade / Downgrade Strategy
+1. The configuration file is available on the API server and the `--authentication-config` flag is set.
+2. No `--oidc-*` flags are set.
```

Comment:
Just to make sure I understand, admins can continue to use `--oidc-*` indefinitely?  But if admins do decide to use `--authentication-config` they must not use `--oidc-*` ?